locale/hu.json

```
@@ -160,9 +160,18 @@
             },
             "player_ids": {
                 "title": "ID mód váltása",
-                "label": "Toggle showing player IDs (and other info) above the head of all nearby players",
-                "alert_show": "Közeli játékosok ID-jének mutatása.",
-                "alert_hide": "Közeli játékosok ID-jének elrejtése."
+                "above_head": {
+                    "name": "Over Head IDs",
+                    "label": "Toggle showing player IDs (and other info) above the head of all nearby players",
+                    "alert_show": "Közeli játékosok ID-jének mutatása.",
+                    "alert_hide": "Közeli játékosok ID-jének elrejtése."
+                },
+                "map_blips": {
+                    "name": "Blips on map",
+                    "label": "Toggle players blips on map as well as their NetIDs.",
+                    "alert_show": "Showing players blips on map.",
+                    "alert_hide": "Hiding players blips on map."
+                }
```

```suggestion
                "name": "Fej feletti ID-k",
                "alert_hide": "Közeli játékosok ID-jének elrejtése."
                    "label": "Játékos ID (és egyéb információk) megjelenítésének kapcsolása az összes közeli játékos feje fölött",
                    "alert_show": "Közeli játékosok ID-jének mutatása.",
                    "alert_hide": "Közeli játékosok ID-jének elrejtése."
                },
                "map_blips": {
                    "name": "Blippek a térképen",
                    "label": "Játékosok blipjeinek, és NetID-nek kapcsolása a térképen.",
                    "alert_show": "Játékos blippek megjelenítve a térképen.",
                    "alert_hide": "Játékos blippek elrejetve a térképen."
                }
```
Hungarian translations added.